branchpro/apps/_dash_app.py

```diff
@@ -150,6 +150,51 @@ def parse_contents(self, contents, filename):
         else:
             return html.Div(['Loaded data from: {}'.format(filename)]), df
 
+    def parse_interval_contents(self, contents, filename):
+        """Load a text (csv) file into a list.
+
+        This method is for loading serial interval data. It expects files to
+        have one column .
+
+        Parameters
+        ----------
+        contents : str
+            File contents in binary encoding
+        filename : str
+            Name of the file
+
+        Returns
+        -------
+        html.Div
+            A div which contains a message for the user.
+        numpy.array
+            An array with the loaded file. If the file load was not
+            successful, it will be None.
+        """
+        content_type, content_string = contents.split(',')
+        _, extension = os.path.splitext(filename)
+
+        decoded = base64.b64decode(content_string)
+        try:
+            if extension in ['.csv', '.txt']:
+                # Assume that the user uploaded a CSV or TXT file
+                serial_interval = pd.read_csv(
+                    io.StringIO(decoded.decode('utf-8'))).values[:, 0]
```

Comment:
The first value in any uploaded interval is being cut off, presumably because here pandas thinks that value is a column title. Try sending `header=None` to the read_csv function.